pkg/cmd/build/build.go

```diff
@@ -89,7 +89,11 @@ func runBuild(opts buildOptions) error {
 			return err
 		}
 
-		tmpDir, err := ioutil.TempDir("/tmp", "wasme")
+		pwd, err := os.Getwd()
+		if err != nil {
+			return err
+		}
+		tmpDir, err := ioutil.TempDir(pwd, "wasme")
```

Comment:
@yuval-k this was the change needed to fix CI; i believe we couldn't mount a directory outside of `/workspace` to the container in CI (though no error appears)

Reply:
use TMPDIR var to fix this in CI